src/people.js

```diff
@@ -1,31 +1,70 @@
-/**
- * @typedef {Object} Person
- * @property {string} name - Person's name
- * @property {string} lastname - Person's lastname
- * @property {number} age - Person's age
- * @property {number} weight - Person's weight
- * @property {number} height - Person's height
- */
-
-/**
- * @typedef {Object} PeopleResponse
- * @property {number} ageProm - People average age
- * @property {number} heightProm - People average height
- * @property {Person} youngerPerson - Younger person information
- * @property {Person} tallerPerson - Taller person information
- */
-
-/** 
- * The height in cm of several people was taken, in addition, the age of each person and 
- * extra information are also known. An algorithm is required to calculate average age and 
- * average height. In addition, we need to know the information of the tallest person, and 
- * the information of the youngest person.
- * 
- * @param {Person[]} people - People information to be processed
- * 
- * @returns {PeopleResponse}  - Processed information
-*/
-
-function peopleInformation(people) { }
-
-module.exports = peopleInformation;
+/**
+ * @typedef {Object} Person
+ * @property {string} name - Person's name
+ * @property {string} lastname - Person's lastname
+ * @property {number} age - Person's age
+ * @property {number} weight - Person's weight
+ * @property {number} height - Person's height
+ */
+
+/**
+ * @typedef {Object} PeopleResponse
+ * @property {number} ageProm - People average age
+ * @property {number} heightProm - People average height
+ * @property {Person} youngerPerson - Younger person information
+ * @property {Person} tallerPerson - Taller person information
+ */
+
+/** 
+ * The height in cm of several people was taken, in addition, the age of each person and 
+ * extra information are also known. An algorithm is required to calculate average age and 
+ * average height. In addition, we need to know the information of the tallest person, and 
+ * the information of the youngest person.
+ * 
+ * @param {Person[]} people - People information to be processed
+ * 
+ * @returns {PeopleResponse}  - Processed information
+*/
+
+const getAgeAverage = arrOfObjects => { 
+    let initialValue = 0;
+    const sum = arrOfObjects.reduce((previousValue, currentValue) => previousValue + currentValue.age, initialValue);
+
+    return Math.round(sum / arrOfObjects.length); 
+}
+
+const getHeightAverage = arrOfObjects => {
+    let initialValue = 0;

```

Comment:
It's advisable to not use `let` unless you're planning on changing the value of the variable. Otherwise always use `const` instead. Although you're use this variable as you acc. The reduce method should make a copy of it instead

Reply:
Yup, definitely missed this one. Thank you so much for the feedback, @josectobar!